src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -103,8 +104,11 @@ public String runKubectlCommand(String command, ImmutableList<String> args)
     Set<String> tempFiles = new HashSet<>();
     try {
       // Set up the kubeconfig file for authentication
-      FilePath kubeConfigFile = workspace.createTempFile(".kube", "config");
+      FilePath tempDir = WorkspaceList.tempDir(workspace);
+      tempDir.mkdirs();
+      FilePath kubeConfigFile = tempDir.createTempFile(".kube", "config");
       tempFiles.add(kubeConfigFile.getRemote());
```

Comment:
You should not need to do that, since `tempDir.deleteRecursive()` would suffice.

Reply:
Done.